fixtures/eurolite/led-z-200-tcl.json

```
@@ -0,0 +1,142 @@
+  },
+  "links": {
+    "manual": [
+      "https://www.steinigke.de/download/51918516-Anleitung-58253-1.100-eurolite-led-z-200-strahleneffekt-de_en.pdf"
+    ],
+    "productPage": [
+      "https://www.steinigke.de/de/mpn51918516-eurolite-led-z-200-strahleneffekt.html"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=BH7UX8sXT7Y",
+      "https://www.steinigke.de/download/51918516-Video-55873-eurolite-led-z-200-strahleneffekt-de_en_es_fr.flv"
+    ]
+  },
+  "physical": {
+    "dimensions": [360, 300, 280],
+    "weight": 3,
+    "power": 40,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED TCL"
+    },
+    "lens": {
+      "degreesMinMax": [125, 125]
+    }
+  },
+  "availableChannels": {
+    "Motor Rotation": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 20],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [21, 170],
+          "type": "Rotation",
+          "speedStart": "slow CCW",
+          "speedEnd": "fast CCW"
+        },
+        {
+          "dmxRange": [171, 255],
+          "type": "Rotation",
+          "speedStart": "slow CW",
+          "speedEnd": "fast CW",
+          "comment": "Clockwise and counter clockwise rotation",
+          "helpWanted": "What does this rotation mean?"
+        }
+      ]
+    },
+    "Program": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 35],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [36, 70],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "helpWanted": "How does the strobe change in this range?"
+        },
+        {
+          "dmxRange": [71, 105],
+          "type": "Effect",
+          "effectName": "Fade",
+          "helpWanted": "How does the fade change in this range?"
+        },
+        {
+          "dmxRange": [106, 140],
+          "type": "Effect",
+          "effectName": "Color-change",
+          "helpWanted": "Which colors are changing at which DMX values?"
+        },
+        {
+          "dmxRange": [141, 175],
+          "type": "Effect",
+          "effectName": "Color-change + strobe",
+          "helpWanted": "Which colors are changing at which DMX values? How does the strobe incorperate with it?"
+        },
+        {
+          "dmxRange": [176, 210],
+          "type": "Effect",
+          "effectName": "Cross-fade",
+          "helpWanted": "Which colors are changing at which DMX values?"
+        },
+        {
+          "dmxRange": [211, 255],
+          "type": "Effect",
+          "effectName": "Cross-fade + strobe",
+          "helpWanted": "Which colors are changing at which DMX values? How does the strobe incorperate with it?"
+        }
+      ]
+    },
+    "Program Speed": {
+      "capability": {
+        "type": "EffectSpeed",
+        "speedStart": "slow",
+        "speedEnd": "fast",
+        "helpWanted": "At which DMX values is the effect disabled?"
```

```suggestion
        "speedEnd": "fast"
```

Whether the effect is enabled or disabled is controlled via channel 2. This channel only controls its speed.